scripts/docker-script.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/bash
+set -e
+cd /main
+OS=$(echo $(awk -F= '$1=="ID" { print $2 ;}' /etc/os-release) | xargs)
+VERSION=$(echo $(awk -F= '$1=="VERSION_ID" { print $2 ;}' /etc/os-release) | xargs)
+
+echo "${OS} ${VERSION} is detected."
+
+install_dependencies_apt()
+{
+    sudo apt-get update
+    sudo apt-get install --fix-broken
+    sudo apt-get install -y wget git gzip tar
+    # Remove the existing libmsquic package and dependencies for Ubuntu
```

Comment:
Are these already here because we're using a .NET that preinstalls them?

Reply:
I'm going to change these images with much cleaner base images